stable/chartmuseum/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+description: Helm Chart Repository with support for Amazon S3 and Google Cloud Storage
+name: chartmuseum
+version: 0.4.0
+appVersion: 0.2.7
+home: https://github.com/chartmuseum/chartmuseum
+icon: https://raw.githubusercontent.com/chartmuseum/chartmuseum/master/logo.png
+maintainers:
+- name: ChartMuseum
+  email: chartmuseum@gmail.com
+- name: Cloud Posse
```

Comment:
Can you change this to be a github id?
Also can you add yourself to the list of maintainers too?

Reply:
Is this a requirement? I'd rather not. I'm seeing others in stable doing this similarly, such as https://github.com/kubernetes/charts/blob/master/stable/rabbitmq/Chart.yaml

Reply:
Are you saying just remove the space in "Cloud Posse" ?
https://github.com/cloudposse

Reply:
@jdolitsky I believe @prydonius meant specifying `name: cloudposse` instead

Reply:
@jdolitsky I think @prydonius meant specifying `name: cloudposse` ??!

Reply:
> Is this a requirement?
Good question, it was something we've discussed in the charts calls, but I need to verify today whether we actually want to make this a hard requirement. In particular:

- OWNERS files should represent specific people (i.e. not orgs) that maintain the chart
- Chart.yaml maintainers should match OWNERS
- Chart.yaml maintainer names should be GitHub IDs (this is so that it's easier to reference maintainers of a chart)

There may be a case for additionally including orgs as maintainers in Chart.yaml (for metadata purposes, not automation/OWNERS). However, these should also be GitHub IDs of orgs (so github.com/cloudposse would be okay).

During today's call I'd like to decide on requirements for this, feel free to join in https://docs.google.com/document/d/1h6UTTuNRbFI81higrN3JUV2XxyzqqVjZET4Xz4WTR-8/edit#.